Follow the project associated with current buffer

katspaugh opened the pull request:
See #36

katspaugh commented:
@sabof I've removed `pe/open` and `pe/project-cache-filename`.

sabof commented on project-explorer.el:
```diff
@@ -1736,6 +1742,45 @@ outside of the project's root."
+        (let ((win (selected-window)))
+          (project-explorer-open)
+          (select-window win)))
+    (pe/stop-follow-current)))
```
You probably want to check `pe/follow-current`, and not `(pe/get-project-explorer-window)` when disabling. What if a user has multiple frames?

katspaugh replied:
@sabof right, I didn't think about multiple frames!

Commit:
Add pe/follow-current option to automatically switch to the project associated with current edit buffer

katspaugh commented:
OK, I added the `pe/follow-current` check, but also left the `(pe/get-project-explorer-window)` check intact to avoid opening project explorer when it's not already opened in the current frame. 

Also squashed commits into one.

sabof commented:
Seems to work, merged

katspaugh commented:
Thanks!

jimeh commented:
:heart: :smile:
